draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -315,6 +315,14 @@ applications running in a third-party browsing context (i.e., a context that
 has a different origin than the top-level browsing context), or a private
 browsing context.
 
+Network Interface Enumeration
+----------------------------
+
+The number of mDNS hostname candidates might provide a fingerprinting dimension.
+This is in particular the case for network interfaces whose existence are not exposed
+through other means like STUN or TURN servers.
+No new information is provided when restricting mDNS hostname candidates to default route only.
```

Comment:
Some estimate of the amount of entropy surfaced here would help readers assess the importance of this finding.

Reply:
Agreed, and hopefully it should be small.
I am unsure how we can evaluate the amount of entropy without doing some experiments there.

The main point though is to say that we should only care about this at the time we will consider exposing non default route candidates.

Reply:
I think this document should probably stick to the technique and let other documents make recommendations about modes, so we probably need to have some short, if hand-wavy, discussion of impact.